### DISCUSSION TITLE
[641320] 烹饪模拟器 Cooking Simulator（本体）

### DISCUSSION THREAD

**Discussion by Dew778** [pjy612/SteamManifestCache#382] [Share]
`
appid：641320
`

```
641321;693af0f9d3395af0e1b880807b3b8b17e6391a03fc860c96b19ed41405f60b47
```

[641320_config.vdf.txt](https://github.com/pjy612/SteamManifestCache/files/13972724/641320_config.vdf.txt)
[[641320] 烹饪模拟器 Cooking Simulator（本体）清单.zip](https://github.com/pjy612/SteamManifestCache/files/13973056/641320.Cooking.Simulator.zip)

Code：7337BCEA697F2EBE
Email:3394748293@qq.com

**Comment by ZEERDEER** [comment]
标题格式 #26

**Reply by Dew778** [reply]
已修改
